samples/JwtBearerSample/Startup.cs

```diff
@@ -59,7 +57,7 @@ public void ConfigureServices(IServiceCollection services)
                 {
                     OnAuthenticationFailed = c =>
                     {
-                        c.HandleResponse();
+                        c.Ignore();
```

Comment:
`Handle` is not ideal because it's easily confused with `HandleResponse`, that is not available for the JWT bearer middleware (since it doesn't handle requests, it can't terminate them, unlike the OIDC middleware).

If you don't like the `Ignore` verb, something like `StopAuthentication()` would likely fit.